sawbuck_app/src/views/accept_offer_modal.js

```diff
@@ -138,13 +138,15 @@ const getAsset = (id, holdings) => {
 const AcceptOfferModal = {
   oninit (vnode) {
     vnode.state.holding = {}
-    vnode.state.acceptance = {}
+    vnode.state.acceptance = {'count': 1}
```

Comment:
This commit should be removed in favor of the fix implemented here:
https://github.com/hyperledger/sawtooth-marketplace/pull/77/commits/585cbe36cb5a5bc7e80c5753d66938b16bd6b8b1

Reply:
Sounds good, ill delete the commit and rename the PR to reflect the bug fix